packages/react-composites/src/composites/CallComposite/adapter/AzureCommunicationCallAdapter.ts

```diff
@@ -302,6 +303,7 @@ export type AdapterStateModifier = (state: CallAdapterState) => CallAdapterState
 export class AzureCommunicationCallAdapter<AgentType extends CallAgent | BetaTeamsCallAgent = CallAgent>
   implements CommonCallAdapter
 {
+  private _logger: AzureLogger;
```

Comment:
This should be working fine, but "communication-react:composite" sounds like a logger running all across the whole composite package, so it would be nice to have a separated file then we are able to import it in any file we want in this repo.
Here is a good example for chat stateful client
/workspaces/communication-ui-library/packages/chat-stateful-client/src/Logger.ts